website/app/components/doc/color-swatch/index.hbs

```diff
@@ -0,0 +1,24 @@
+
+  <div class="doc-color-swatch">
+    <div class="doc-color-swatch__preview doc-color-swatch__preview-{{this.cssHelper}}" style={{this.cardStyle}}></div>
+    <div class="doc-color-swatch__info">
+      <h3 class="doc-color-swatch__name">{{this.colorName}}</h3>
+      <ul class="doc-color-swatch__list" role="list">
+        <li class="doc-color-swatch__listitem">
+          <span class="doc-color-swatch__listitem--context">CSS Variable</span>
```

Comment:
Non-blocking: Technically, this is not a BEM modifier but an element, so to my eye, it would be more of a `doc-color-swatch__listitem-context`